pil_analyzer/src/display.rs

```diff
@@ -211,12 +211,14 @@ namespace T(65536);
     col witness instr_dec_CNT;
     col witness instr_assert_zero;
     (T.instr_assert_zero * (T.XIsZero - 1)) = 0;
+    col witness X;
```

Comment:
Why did this change?

Reply:
Because we now check if referenced columns exist. This was referenced but didn't exist.